Xamarin.Forms.Core/Label.cs

```diff
@@ -215,13 +217,19 @@ public Thickness Padding
 			get { return (Thickness)GetValue(PaddingProperty); }
 			set { SetValue(PaddingProperty, value); }
 		}
-		
+
 		public TextType TextType
 		{
 			get => (TextType)GetValue(TextTypeProperty);
 			set => SetValue(TextTypeProperty, value);
 		}
 
+		public bool AutoFit
```

Comment:
I feel we need to tweak the name a little. 

AutoFitText? AutoFitFont? Or maybe something completely different like FitTextToWidth

Reply:
I think `AutoFitText` is better, because the text adjusts to bounds of view (if mult-line is allowed)